backend/app/adapter/grpcapi/metatag.go

```diff
@@ -0,0 +1,48 @@
+package grpcapi
+
+import (
+	"context"
+
+	"github.com/short-d/short/backend/app/adapter/grpcapi/proto"
+	"github.com/short-d/short/backend/app/usecase/shortlink"
+)
+
+// MetaTagServer represents MetaTag gRPC server
+type MetaTagServer struct {
+	metaTag shortlink.MetaTag
+}
+
+var _ proto.MetaTagServiceServer = (*MetaTagServer)(nil)
+
+// GetOGTags fetches Open Graph Tags from persistent storage given alias
+func (m MetaTagServer) GetOGTags(ctx context.Context, req *proto.GetOGTagsRequest) (*proto.GetOGTagsResponse, error) {
+	ogMetaTags, err := m.metaTag.GetOpenGraphTags(req.GetAlias())
+	if err != nil {
+		return &proto.GetOGTagsResponse{}, err
+	}
+
+	return &proto.GetOGTagsResponse{
+		Title:       *ogMetaTags.Title,
+		Description: *ogMetaTags.Description,
+		ImageUrl:    *ogMetaTags.ImageURL,
```

Comment:
All the fields are optional. We need to check those in order to make sure we are not crashing the server when they are nil